go-controller/pkg/clustermanager/secondary_network_cluster_manager.go

```diff
@@ -56,9 +95,13 @@ func (sncm *secondaryNetworkClusterManager) Stop() {
 func (sncm *secondaryNetworkClusterManager) NewNetworkController(nInfo util.NetInfo,
 	netConfInfo util.NetConfInfo) (nad.NetworkController, error) {
 	topoType := netConfInfo.TopologyType()
+	networkId, err := sncm.networkIDAllocator.allocateID(nInfo.GetNetworkName())
```

Comment:
allocate id only if the topology is supported?

Reply:
Done